superset/assets/javascripts/SqlLab/components/RunQueryActionButton.jsx

```diff
@@ -43,6 +44,7 @@ export default function RunQueryActionButton(props) {
       onClick={() => props.runQuery(true)}
       key="run-async-btn"
       tooltip={t('Run query asynchronously')}
+      disabled={!props.sql.trim()}
```

Comment:
I'm confused as to why the linter isn't asking for `sql` to be in `propTypes`. I wanted to make sure it's entered as a `string` and can never be `null` through a defaultProps = ''

Reply:
I have made changes, please review :+1: